client/src/pages/Login/Login.tsx

```diff
@@ -37,22 +37,26 @@ export default function Login(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
+    <Grid container component="main" className={classes.root} justify="center" alignItems="center">
       <CssBaseline />
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
           <Box width="100%" maxWidth={450} p={3} alignSelf="center">
             <Grid container>
               <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
+                <Typography className={classes.welcome} component="h1" variant="h5" align="center">
+                  Login
                 </Typography>
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <Typography className={classes.reminder} component="div" variant="body1" align="center">
```

Comment:
```suggestion
            <Typography component={Link} to="/signup" className={classes.reminder} component="div" variant="body1" align="center">
```
You can combine MUI's Typography with react-router Link. Ditto where applicable :)